### PR TITLE
Move potparams to and organized real_world_constants

### PR DESCRIPTION
**Pull request by DrDab**

### CODE REVIEW

File: src/world_interface/real_world_constants.h [hunk 1]
```diff
@@ -29,6 +29,39 @@ constexpr auto pidMotors = frozen::make_unordered_set<motorid_t>(
 	{motorid_t::armBase, motorid_t::shoulder, motorid_t::elbow, motorid_t::forearm,
 	 motorid_t::wrist});
 
+// struct PotenetiometerParams => potparams_t
```
Comment (abhaybd) [inline src/world_interface/real_world_constants.h:32]: Remove this line pls

File: src/world_interface/real_world_constants.h [hunk 2]
```diff
@@ -29,6 +29,39 @@ constexpr auto pidMotors = frozen::make_unordered_set<motorid_t>(
...
+	int32_t mdeg_lo;
+	/** The "high" point on the ADC scale. */
+	uint16_t adc_hi;
+	/** The "high" point on the joint rotation scale. */
+	int32_t mdeg_hi;
+};
+
+struct encparams_t {
+	bool isInverted;
+	int pulses_per_joint_revolution;
```
Comment (abhaybd) [inline src/world_interface/real_world_constants.h:53]: Abbreviate to ppjr? And add a comment saying what that means.

File: src/world_interface/real_world_constants.h [hunk 3]
```diff
@@ -29,6 +29,39 @@ constexpr auto pidMotors = frozen::make_unordered_set<motorid_t>(
...
+};
+
+constexpr auto encMotors = frozen::make_unordered_map<motorid_t, encparams_t>(
+	{{motorid_t::shoulder, {.isInverted = false, .pulses_per_joint_revolution = 20 * 1000}},
+	 {motorid_t::elbow, {.isInverted = true, .pulses_per_joint_revolution = 36 * 1000}}});
```
Comment (abhaybd) [inline src/world_interface/real_world_constants.h:58]: You have the numbers to calculate the ppjrs, can you put them here instead of these dummy values?

**Review by abhaybd — CHANGES_REQUESTED** [review]
Mostly good, just some minor changes needed

File: src/world_interface/real_world_interface.cpp [hunk 4]
```diff
@@ -67,13 +67,35 @@ void initMotors() {
 	can::motor::initMotor(motorSerialIDMap.at(motorid_t::rearLeftWheel));
 	can::motor::initMotor(motorSerialIDMap.at(motorid_t::rearRightWheel));
 
+	// loop over potMotors and init accordingly with can::motor::initPotentiometer
+	// loop over encMMotors and init accordingly
```
Comment (abhaybd) [inline src/world_interface/real_world_interface.cpp:71]: These comments are a bit superfluous, we can remove them, since they just describe a for loop.

**Review by abhaybd — CHANGES_REQUESTED** [review]
Looks good! Minor nitpick, and also can you merge the latest changes from master? Then this should be good to merge.

**Comment by DrDab** [comment]
@abhaybd Rm'd unnecessary comments and merged master branch changes

**Review by abhaybd — APPROVED** [review]
Looks good!

**Comment by abhaybd** [comment]
Good job getting this turned around so quickly!

**Pull request merged**